stable/datacube-ows/templates/ows-deployment.yaml

```diff
@@ -138,6 +139,15 @@ spec:
 {{- if or .Values.owsConfig.configMap .Values.owsConfig.image }}
         - mountPath: {{ .Values.owsConfig.path }}
           name: ows-config-volume
+{{- end }}
+{{- if .Values.pyspy.image }}
```

Comment:
why we require two variables to configure this settings - `pyspy` and `pyspyImage` ?

Reply:
can easily be achieve by one variable

Reply:
ok, changed the way another section is done as well, merged image block into `ows` value